pycodestyle.py

```diff
@@ -1545,7 +1554,7 @@ def ambiguous_identifier(logical_line, tokens):
                 ident = text
                 pos = start
         # function parameter definitions
-        if is_func_def:
+        if is_func_def or (is_lambda_def and prev_text in ('lambda', ',')):
```

Comment:
I think this is still not quite right? `lambda x=a.I: None`

Reply:
trying to follow -- that example shouldn't trigger an error for `a.I`, right? in the current implementation it wouldn't because the prior token isn't `lambda` or `,`

Reply:
oh hmmm, then how about `lambda *i: None` or `lambda **i: None` (I think this already properly handles `lambda *, i: None` and `lambda /, i: None` probably)